JamitFoundationExample/JamitFoundationExample.xcodeproj/project.pbxproj

```diff
@@ -977,7 +1008,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = DB7EUU9D79;
 				INFOPLIST_FILE = App/SupportingFiles/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
There are 3 places in the sample project where there are still `if #available` checks that are presumably no longer needed with this deployment target bump. Could you remove them?